packages/bot/src/components/notification-messages.jsx

```diff
@@ -0,0 +1,35 @@
+import classNames    from 'classnames';
+import React         from 'react';
+import { PropTypes } from 'prop-types';
+import { connect }   from '../stores/connect';
+import                    '../assets/sass/notification-messages.scss';
+
+const NotificationMessages = ({
```

Comment:
@zaki-binary is there a reason other than position to create this component?
cause we have a css-variable that give you the value of run-panel-width. its zero when run-panel is close and 366 when its open, so i believe you can use that to override the ` .notification-messages` position.

Reply:
I think it's cleaner to handle this in it's own sort of wrapper component, since: -
1) This component directly connects to `UIStore` on it's own and we only want it to render if there are existing `notification messages` in the `UIStore`. I initially did it in `workspace.jsx` but I think it should be it's own component since `workspace` isn't connected to any stores.

2) We can handle what to do in the state changes (if we end up having anymore in the future) here within the component and it can use it's own `sass` file for the `modifier` states, no need to modify other existing files if we ever have to update it in the future.